helm/ako/values.yaml

```diff
@@ -127,7 +127,7 @@ rbac:
   # Creates the pod security policy if set to true
   pspEnable: false
 
-
+# If username and either password or authtoken are not specified, avi-secret will not be created. AKO will assume that the avi-secret already exists and will reference it.
```

Comment:
I think it will be good to mention this: `In such scenario(where username and pass/authtoken empty),  value of `certificateAuthorityData` will be taken from existing `avi-secret`.
Please re-word as per your convenience.

Reply:
I have updated the PR with these changes.